moped-editor/src/views/projects/projectView/ProjectComponents/EditModeDialog.js

```diff
@@ -53,15 +61,17 @@ const EditModeDialog = ({ showDialog, editDispatch, onEditFeatures }) => {
             </Button>
           </Grid>
           <Grid item>
-            <Button
-              variant="outlined"
-              color="primary"
-              className={classes.margin}
-              startIcon={<TimelineIcon />}
-              onClick={onEditFeatures}
-            >
-              Map
-            </Button>
+            {!isSignalComponent && (
```

Comment:
@mddilley this works just fine—but i'm curious what you think it'd look like to skip this dialog entirely (or immediately fire off `onEditAttributes`) rather than having the user click the "Attributes" button

Reply:
Oooo yeah, that would be a good one because it will feel faster and look less awkward for sure. 👍

Reply:
i like this idea!